h/accounts/schemas.py

```diff
@@ -246,11 +246,11 @@ class ResetPasswordSchema(CSRFSchema):
     # call it `user` because when validated, it will return a `User` object.
     user = colander.SchemaNode(
         ResetCode(),
-        title=_('Your reset code:'),
+        title=_('Reset code'),
```

Comment:
I think you missed my point here, which was that you haven't made equivalent changes for two of the four forms.